### DISCUSSION TITLE
CLI - "add-module" to Blazor App

### DISCUSSION THREAD

**Discussion by 274188A** [abpframework/abp#6591] [Q&A] [answered]
Hi,

I am using the following to try and add a module (already exists)  to a blazor ui app:

```
abp add-module setting-management
```

I'm getting:-

>ERROR: 'setting-management' module could not be found!

I have tried the preview and nightly channels with no luck - as well as many permutations of the module name.
I can see the source code in the repo - maybe the nuget package is not there yet?

Thanks,
John

**Comment by realLiangshiwei** [comment, marked as answer]
Try 
`abp add-module Volo.SettingManagement`

**Reply by 274188A** [reply]
all good except i expected to see the menu added.  If it was not added after the add-module what are my options?

**Reply by Stirda** [reply]
Menu exists only if a setting contributor exists too. ABP Blazor has none yet. You have to create at least one by yourself. See https://github.com/abpframework/abp/issues/5541#issuecomment-703782693

**Reply by 274188A** [reply]
I also see somebody has done this for [MVC](https://github.com/EasyAbp/Abp.SettingUi) already - thanks for the tip!
